### PR TITLE
Add tsx and css homepage 

### PR DESCRIPTION
**Pull request by lpstrevisan**

### CODE REVIEW

File: frontend/src/app/home/pages/homePage/index.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        <header className="header">
+          <div className="logo">
+            <img src="logo-icon.jpg" alt="Logo AlmoCIn" className="logo-icon"/>
+            <h1> AlmoCIn </h1>
+          </div>
+  
+          <div>
+            <button className="menu-buttons"> Cardápio </button>
+            <button className="menu-buttons"> Histórico de Pedidos </button>
+            <button className="menu-buttons"> Estatísticas do Usuário </button>
```
Comment (JDaniielC) [inline frontend/src/app/home/pages/homePage/index.tsx]: User Statistics doesn't exists.

**Review by JDaniielC — CHANGES_REQUESTED** [review]
You'd improve this page following the design (search for first canva project)

File: frontend/src/app/home/components/baseLayout/index.tsx
Comment (JDaniielC) [inline frontend/src/app/home/components/baseLayout/index.tsx:1]: It's better move baseLayout for shared and create menuItems for each module (home, admin)

File: frontend/src/app/itemMenu/components/baseLayout/index.tsx
Comment (JDaniielC) [inline frontend/src/app/itemMenu/components/baseLayout/index.tsx:1]: Again, reuse code.

File: frontend/src/app/itemMenu/context/itemMenuContext/index.tsx
Comment (JDaniielC) [inline frontend/src/app/itemMenu/context/itemMenuContext/index.tsx:1]: menuContext already exists.

**Review by JDaniielC — CHANGES_REQUESTED** [review]
Refactor baseLayout reuse and remove menuContext

File: frontend/src/app/home/components/baseLayout/index.tsx
Comment (lpstrevisan) [reply frontend/src/app/home/components/baseLayout/index.tsx:1]: i didn't understand "create menuItems for each module", there is already a "menu" in admin module, should I just move the itemMenu to home module?

**Review by JDaniielC — APPROVED** [review]

**Pull request merged**